src/routes/jira/server/jira-app-creation-post.ts

```diff
@@ -0,0 +1,24 @@
+import e, { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraAppCreationPost = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<e.Response<number, Record<string, string>> | void> => {
+	const { gheServerURL, installationId } = req.body;
+
+	try {
+		const gitHubServerApps = await GitHubServerApp.getAllForGitHubBaseUrl(gheServerURL, installationId);
+
+		if (gitHubServerApps && gitHubServerApps?.length > 0) {
```

Comment:
What am I doing wrong here? If I just use the ternary the linter bitches that `gitHubServerApps` might be undefined 🤨 So I added the additional check but... shouldnt the ? work????

Reply:
I swear we'll fixup that b**tch sometime .

Reply:
Its because `getAllForGitHubBaseUrl` can return a `null` value. 
One way could be to return a blank array instead of null for  this condition `(!gitHubBaseUrl || !installationId)` in `getAllForGitHubBaseUrl`, which would make the return value to always be an array. Then this `gitHubServerApps` check can be avoided and can simply use `gitHubServerApps.length`.

Reply:
just try `if (gitHubServerApps?.length) {`

Reply:
I'm also not seeing any linting issues in the build for this... could be just your local IDE doing something weird?